File: src/main/java/com/damianryan/octopus/model/SampleQuote.java
package com.damianryan.octopus.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AccessLevel;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class SampleQuote {

    @JsonProperty("electricity_single_rate")
    AnnualCost electricitySingleRate;

    @JsonProperty("electricity_dual_rate")
    AnnualCost electricityDualRate;

    @JsonProperty("dual_fuel_single_rate")
    AnnualCost dualFuelSingleRate;

    @JsonProperty("dual_fuel_dual_rate")
    AnnualCost dualFuelDualRate;
}

File: src/main/kotlin/com/damianryan/octopus/model/SampleQuote.kt
package com.damianryan.octopus.model

import com.fasterxml.jackson.annotation.JsonProperty

data class SampleQuote(
    @JsonProperty("electricity_single_rate") val electricitySingleRate: AnnualCost? = null,
    @JsonProperty("electricity_dual_rate") val electricityDualRate: AnnualCost? = null,
    @JsonProperty("dual_fuel_single_rate") val dualFuelSingleRate: AnnualCost? = null,
    @JsonProperty("dual_fuel_dual_rate") val dualFuelDualRate: AnnualCost? = null,
)